client/src/components/Posts.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {connect} from 'react-redux';
+
+import {getPosts} from '../actions';
+
+
+
+class Posts extends React.Component {
+
+    state= {
+        posts: []
+    };
+
+    componentDidMount() {
+        this.props.getPosts();
+    }
+
+
+    render() {
+        return (
+            <div className="SmurfList">
```

Comment:
I think they're hobbits and not smurfs ;)

Reply:
Ha! And whoops. This is also fixed.